### PR TITLE
[PAPI-194] Rename to mining governance

### PR DESCRIPTION
**Pull request by drmathias**

### CODE REVIEW

**Comment by drmathias** [comment]
Still need to run these locally. Will resolve this comment once that's done and confirmed it runs smoothly.

File: scripts/PAPI-194_MiningGovernanceNaming.sql [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ END;
+//
+
+CALL MiningGovernanceNaming();
+//
+
+DROP PROCEDURE MiningGovernanceNaming;
+//
+
+DELIMITER ;
```
Comment (drmathias) [inline scripts/PAPI-194_MiningGovernanceNaming.sql:71]: Still need to run these locally. Will resolve this comment once that's done and confirmed it runs smoothly.

Comment (drmathias) [reply scripts/PAPI-194_MiningGovernanceNaming.sql:68]: Made some amendments and successfully runs locally

**Review by mrtpain — APPROVED** [review]
lgtm, nice 👍

**Pull request merged**